pom.xml

```diff
@@ -788,6 +796,12 @@
                 <version>4.2.37</version>
             </dependency>
 
+            <dependency>
+                <groupId>io.github.luohao</groupId>
+                <artifactId>fastlanes-java</artifactId>
+                <version>1</version>
+            </dependency>
```

Comment:
We avoid defining dependencies in root pom.xml when it is used from a single module.